docs/usage.md

```diff
@@ -241,6 +241,10 @@ Tool for clustering CASR reports
       -V, --version
               Print version
 
+If you set `CASR_CLUSTER_UNIQUE_CRASHLINE` environment variable as "1"
```

Comment:
maybe we should just check that env is set/unset

Reply:
IMHO, explicit value is better. Some DevOps code may just do ENV_VAR=0